src/api-umbrella/web-app/app/controllers/api/v1/users_controller.rb

```diff
@@ -49,20 +49,25 @@ def create
 
     respond_to do |format|
       if(@api_user.save)
-        send_email = (params[:options] && params[:options][:send_welcome_email].to_s == "true")
+        send_welcome_email = (params[:options] && params[:options][:send_welcome_email].to_s == "true")
+        send_notify_email = (params[:options] && params[:options][:send_notify_email].to_s == "true")
```

Comment:
Is there a specific code to write to get data from `ApiUmbrellaConfig[:send_notify_email]`?